src/components/Footer.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+import { Filter } from '../types/Filter';
+
+interface Props {
+  todos: Todo[];
+  filter: Filter;
+  setFilter: (filter: Filter) => void;
+  onClearCompleted: () => void;
+  // tempTodo: Todo | null;
+  // isLoading: boolean;
+}
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  filter,
+  setFilter,
+  onClearCompleted,
+  // tempTodo,
+  // isLoading,
+}) => {
+  const incompleteTodoCount = todos.filter(todo => !todo.completed);
+
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        {`${incompleteTodoCount.length} items left`}
+      </span>
+
+      {/* Active filter should have a 'selected' class */}
+      <nav className="filter">
+        <a
+          href="#/"
+          className={classNames(
```

Comment:
Also, you can make it name short
```suggestion
          className={cn(
```